src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  margin-bottom: 46px;
+  max-width: $cards-width-size;
+  background: $color-white;
+
+  &__img {
+    display: flex;
+    justify-content: center;
+    padding-top: 32px;
+  }
+
+  &__imac {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text {
+    display: flex;
+    align-self: center;
+    margin: 0;
+    padding-top: 40px;
+    width: 166px;
+    font-size: $small-font-size;
+    line-height: $medium-line-height;
+    font-weight: $medium-font-weight;
+    color: $color-light-black;
+  }
+
+  &__code {
+    @extend %small-text;
+
+    margin: 0;
+    padding-top: 4px;
+    color: $color-secondary-grey;
+  }
+
+  &__stars-reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    padding-top: 16px;
+  }
+
+  &__review {
+    @extend %small-text;
+
+    margin: 0;
+    color: $color-light-black;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    padding-top: 24px;
+  }
+
+  &__price-label {
+    margin: 0;
+    font-size: $small-font-size;
+    line-height: $medium-line-height;
+    font-weight: $small-font-weight;
+    color: $color-secondary-grey;
+  }
+
+  &__price-value {
+    margin: 0;
+    font-size: 16px;
+    line-height: $medium-line-height;
+    font-weight: $big-font-weight;
+    color: $color-light-black;
+  }
+
+  &__button {
+    padding: 16px 0;
+  }
+
+  &__button-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid;
+    border-radius: 5px;
+    width: 166px;
+    font-size: 14px;
+    line-height: 38px;
+    font-weight: $big-font-weight;
+    background-color: $color-accent-blue;
+    text-decoration: none;
+    text-align: center;
+    text-transform: uppercase;
+    color: $color-white;
+  }
+
+  &__button-buy:hover {
+    border-radius: 5px;
+    border-color: $color-accent-blue;
+    color: $color-accent-blue;
+    background: $color-white;
+  }
+}
+
+.card:not(:nth-of-type(4n)) {
+  margin-right: $cards-margin-right;
+}
```

Comment:
Do not apply any external geometry directly to the BEM block use BEM mixes instead

Reply:
External geometry stands for margins only?

Reply:
Done

Reply:
> External geometry stands for margins only?
all that could affect the positioning of the block and interfere its reusability